index.html

```diff
@@ -604,6 +616,149 @@ <h3>Set the session description</h3>
       </ol>
     </section>
   </section>
+  <section id="rtcicetransport">
+    <h3>
+      {{RTCIceTransport}} extensions
+    </h3>
+    <p>
+      The {{RTCIceTransport}} interface is defined in [[WEBRTC]]. This document extends that interface to allow an
+      application to observe and affect certain actions that an <dfn>ICE agent</dfn> [[RFC5245]] performs.
+    </p>
+    <p>
+      When the [= ICE agent =] has selected a candidate pair to prune, the [= user agent =] MUST [= queue a task =] to <dfn id="rtcicetransport-propose-prune">propose a candidate pair pruning</dfn>:
+    </p>
+    <ol class="algorithm">
+      <li>
+        <p>
+          Let |connection:RTCPeerConnection| be the {{RTCPeerConnection}} object associated with this [= ICE agent =].
+        </p>
+      </li>
+      <li>
+        <p>
+          If <var>connection</var>.{{RTCPeerConnection/[[IsClosed]]}} is
+          <code>true</code>, abort these steps.
+        </p>
+      </li>
+      <li>
+        <p>
+          Let |transport:RTCIceTransport| be the {{RTCIceTransport}} object associated with this candidate pair.
+        </p>
+      </li>
+      <li>
+        <p>
+          Let |candidatePair:RTCIceCandidatePair| be the candidate pair which is being proposed for pruning.
+        </p>
+      </li>
+      <li>
+        <p>
+          Let |accepted:boolean| be the result of [= fire an event | firing an event =] named
+          {{RTCIceTransport/candidatepairpruneproposed}} at |transport|, using {{RTCIceCandidatePairEvent}}, with the
+          {{Event/cancelable}} attribute initialized to <code>true</code>, and the {{RTCIceCandidatePairEvent/candidatePair}}
+          attribute initialized to |candidatePair|.
+        </p>
+      </li>
+      <li>
+        <p>
+          If |accepted| is <code>false</code>, instruct the [= ICE agent =] to not prune the candidate pair indicated by |candidatePair|, and instead continue to send and respond to ICE connectivity checks on the candidate pair as before.
+        </p>
+      </li>
+      <li>
+        <p>
+          Otherwise, instruct the [= ICE agent =] to prune the candidate pair indicated by |candidatePair|.
+        </p>
+      </li>
+    </ol>
+    <pre class="idl">
+      partial interface RTCIceTransport {
+        attribute EventHandler oncandidatepairadded;
+        attribute EventHandler oncandidatepairpruneproposed;
```

Comment:
I think we should simplify and call it `oncandidatepairprune`. E.g. we don't say `onclickproposed`.